open_city_profile/tests/conftest.py

```diff
@@ -102,3 +104,13 @@ def gql_schema(rf, anon_user_gql_client):
 def mock_responses():
     with responses.RequestsMock(assert_all_requests_are_fired=False) as mock_resps:
         yield mock_resps
+
+
+def unix_timestamp_now():
+    epoch = datetime(1970, 1, 1, tzinfo=timezone.utc)
+    return int((datetime.now(tz=timezone.utc) - epoch).total_seconds())
```

Comment:
Is there a reason datetime.utcnow().timestamp() is not sufficient?

Reply:
`datetime.utcnow()` returns a naive `datetime`. Calling `timestamp()` for naive `datetime`s treats the `datetime` to be local time. That would result in incorrect values (unless local time happens to be UTC).

But `datetime.now(tz=timezone.utc).timestamp()` should actually work.

Reply:
Ah. `time.time()` seems to be the better choice.

Reply:
Changed in a8d5cd162c.

Reply:
`time.time()` doesn't necessarily use the unix epoch so it may not return the unix timestamp.